src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,198 @@
+$blue: #04529D;
+$outer-circle-left-pos: 91%;
+$outer-circle-top-pos: -2%;
+
+$hex-circle-left-pos: -8%;
+$hex-circle-top-pos: -4%;
+
+$intro-color: #00F977;
+$malaria-color: #EB5A61;
+$med-color: #017E84;
+$pc-policy-color: #F2AE3E;
+$white-color: #FFFFFF;
+
+$activity-incomplete: #BEBEBE;
+
+$malaria-outer-circle-color: #E13946;
+$med-outer-circle-color: #07717E;
+$pc-outer-circle-policy-color: #F09938;
+
+$header-blue: #04529D;
+
+@mixin rotation($angle) {
+    -ms-transform: rotate($angle); /* IE 9 */
+    -webkit-transform: rotate($angle); /* Chrome, Safari, Opera */
+    transform: rotate($angle);
+}
+
+@mixin circle($diameter) {
+    display: flex;
+    height: $diameter;
+    min-height: $diameter;
+    width: $diameter;
+    min-width: $diameter;
+    border-radius: 50%;
+    border: 1px solid;
+    border-color: $activity-incomplete;
+}
+.menu-button {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+
+    .hex {
+        z-index: 1;
+        width: auto;
+        height: 20%;
+    }
+
+    .outer-circle {
+        @include circle(85px);
+        position: relative;
+        top: 1em;
+
+        .inner-circle {
+            position: relative;
+            @include circle(52px);
+            margin: auto;
+            align-items: center;
+            div {
+                margin: auto;
+            }
+
+            .arc {
+                overflow: hidden;
+                position: absolute;
+                top: -1em;
+                right: 50%;
+                bottom: 50%;
+                left: -1em;
+                transform-origin: 100% 100%;
+                transform: rotate(0deg) skewX(-1deg);
+                &:before {
+                    content: '';
+                    transform: skewX(1deg);
+                    @include circle(200%);
+                    border-width: 8px;
+                    box-sizing: border-box;
+                    display: block;
+                }
+
+                &.second {
+                    transform: rotate(120deg) skewX(-1deg);
+                }
+                &.third {
+                    transform: rotate(240deg) skewX(-1deg);
+                }
+                &.completed:before {
+                    border-color: $intro-color;
+                }
+            }
+        }
+    }
+}
+
+
+.menu {
+    margin: 20px;
+}
+
+.intro-outer-circle {
+    @extend .outer-circle;
+    background: $white-color;
+}
+
+.intro-inner-circle {
```

Comment:
why can't you just have `class="inner-circle intro-inner-circle"` in the DOM & remove the `@extend` here? you aren't really taking advantage of CSS here